src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");

```

Comment:
Similarly to the start message, the stop message 'Excavator stopped working' could be more informative. Reflect what the Excavator was doing before it stopped, such as 'Excavator has finished digging and is now off.'